### PR TITLE
Website / About page spacing tweaks for mobile

### PR DESCRIPTION
**Pull request by DavidBachmann**
# About page spacing tweaks for mobile
https://app.asana.com/0/1183498341031105/1195547281923255

## What

- AboutPage mobile spacing tweaks
- Make Timeline escape grid on mobile 
- Give eyebrow bigger line-height on mobile.
- General improvements

## Screenshots

Before:
image

After:
image

Before:
image

After:
image

Etc...

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

**Review by barabrian — DISMISSED** [review]
LGTM!

File: apps/web/screens/AboutPage/AboutPage.tsx [hunk 1]
```diff
@@ -160,7 +160,7 @@ const PageHeader: FC<PageHeaderProps> = ({ page }) => {
   return (
     <Background id={slice.id} theme={page.theme}>
       {!!mobileNavigation.length && (
-        <Box display={['block', 'block', 'block', 'none']} paddingBottom={4}>
+        <Hidden above="md">
```
Comment (barabrian) [inline apps/web/screens/AboutPage/AboutPage.tsx:163]: Are we removing padding bottom here? and is it not on the agenda to deprecate Hidden component?

**Review by barabrian — APPROVED** [review]
still looks good to me!

File: apps/web/screens/AboutPage/AboutPage.tsx (hunk 1 above)
Comment (DavidBachmann) [reply apps/web/screens/AboutPage/AboutPage.tsx:163]: Padding bottom was just introducing spacing on top of the page and not affecting the DrawerMenu. Not sure what the point was.

I'll replace Hidden when/if Box supports hiddenAbove syntax.

**Pull request merged**